lib/BBTools/utils/BBMapRunner.py

```diff
@@ -298,7 +357,7 @@ def _build_file_report(self, output_dir, run_log):
         result_file = os.path.join(output_dir, 'bbmap_report.zip')
         with zipfile.ZipFile(result_file, 'w', zipfile.ZIP_DEFLATED, allowZip64=True) as report_zip:
             for file_name in file_list:
-                if file_name.endswith('.gz') or file_name.endswith('.fastq'):
+                if file_name.endswith('.gz') or file_name.endswith('.fastq') or file_name.endswith('.FASTQ'):
```

Comment:
try letting `file_name = file_name.lower()`, then you can skip the caps (would also capture .GZ files, not that those will be common.

Reply:
good point.  as a mea culpa, I always forget which methods modify the item itself, and which return a value without modification, so whenever I use them I have to google it yet again.  I was lazy to google and read and instead put more work into adding too much logic.  busted.